src/Components/DropDown.tsx

```diff
@@ -0,0 +1,83 @@
+import debounce from 'lodash.debounce';
+import { useCallback, useMemo, useState } from 'react';
+import { Person } from '../types/Person';
+
+type DropDownProps = {
+  people: Person[],
+  onSelected?: (person: Person) => void
+};
+
+export const DropDown: React.FC<DropDownProps> = ({
+  people,
+  onSelected = () => { },
+}) => {
+  const [active, setActive] = useState(false);
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
```suggestion
  const [isActive, setIsActive] = useState(false);
```